crates/oxc_minifier/tests/mod.rs

```diff
@@ -11,8 +11,15 @@ use oxc_minifier::{CompressOptions, Minifier, MinifierOptions, PrinterOptions};
 use oxc_span::SourceType;
 
 pub(crate) fn test(source_text: &str, expected: &str) {
-    let source_type = SourceType::default();
+    // let source_type = SourceType::default();
     let options = MinifierOptions { mangle: false, ..MinifierOptions::default() };
+    test_with_options(source_text, expected, options);
+    // let minified = Minifier::new(source_text, source_type, options).build();
+    // assert_eq!(expected, minified, "for source {source_text}");
```

Comment:
Nit: these comments seem like can be removed.

Reply:
done